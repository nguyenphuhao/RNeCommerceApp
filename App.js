/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow strict-local
 */

import React from 'react';
import { createStore } from 'redux';
import { Provider } from 'react-redux';
import combineReducers from './src/reducers';
import LoginScreen from './src/screens/login/LoginScreen';

const initialState = {};
const store = createStore(combineReducers, initialState);

class App extends React.Component {
  render() {
    return (
      <Provider store={store}>
        <LoginScreen />
      </Provider>
    );
  }
}

export default App;
