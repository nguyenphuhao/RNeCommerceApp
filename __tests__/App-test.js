/**
 * @format
 */

import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import App from '../App';

const shallowComponent = () => {
  const wrapper = shallow(<App />);
  return wrapper;
};

describe('RegistrationScreen exists components', () => {
  test('RegistrationScreen exists components - test snapshot', () => {
    const wrapper = shallowComponent();
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
