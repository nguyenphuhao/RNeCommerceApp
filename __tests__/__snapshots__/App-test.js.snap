// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RegistrationScreen exists components RegistrationScreen exists components - test snapshot 1`] = `
<Provider
  store={
    Object {
      "@@observable": [Function],
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
    }
  }
>
  <LoginScreen />
</Provider>
`;
